imaging/loaders/commonLoader.ts

```diff
@@ -273,3 +269,80 @@ export const getSopInstanceUIDFromLarvitarManager = function (
     key => series.instanceUIDs[key] === imageId
   );
 };
+const getTemporalSeriesData = function( series: Series,) {
```

Comment:
Questa funzione la metterei in imageRendering dove c'è getSeriesData, è una funzione interna che sarà usata solo li